apps/console/src/features/roles/components/edit-role/edit-role-basic-updated.tsx

```diff
@@ -0,0 +1,269 @@
+                        name="roleName"
+                        inputType="identifier"
+                        required={ true }
+                        readOnly={ isReadOnly }
+                        value={ role?.displayName }
+                        label={ t("console:manage.features.roles.edit.basics.fields.roleName.name") }
+                        placeholder={
+                            t("console:manage.features.roles.edit.basics.fields.roleName.placeholder")
+                        }
+                        maxLength={ RoleConstants.ROLE_NAME_MAX_LENGTH }
+                        minLength={ RoleConstants.ROLE_NAME_MIN_LENGTH }
+                    />
+                    <Field.Button
+                        form={ FORM_ID }
+                        buttonType="primary_btn"
+                        ariaLabel="Update button"
+                        name="update-button"
+                        hidden={ isReadOnly }
+                        disabled={ isSubmitting }
+                        loading={ isSubmitting }
+                        data-componentid={ `${ componentid }-role-update-button` }
+                        label={ t("extensions:develop.apiResource.tabs.general.form.updateButton") }
+                    />
+                </Form>
+            </EmphasizedSegment>
+            <Divider hidden />
+            {
+                !isReadOnly && (
+                    <DangerZoneGroup sectionHeader="Danger Zone">
+                        <DangerZone
+                            actionTitle={ 
+                                t("console:manage.features.roles.edit.basics.dangerZone.actionTitle",
+                                    { type: "Role" })
+                            }
+                            header={
+                                t("console:manage.features.roles.edit.basics.dangerZone.header",
+                                    { type: "role" })
+                            }
+                            subheader={
+                                t("console:manage.features.roles.edit.basics.dangerZone.subheader",
+                                    { type: "role" })
+                            }
+                            onActionClick={ () => setShowDeleteConfirmationModal(!showRoleDeleteConfirmation) }
+                            data-componentid={ `${ componentid }-role-danger-zone` }
+                        />
+                    </DangerZoneGroup>
+                )
+            }
+            {
+                showRoleDeleteConfirmation && (
+                    <ConfirmationModal
+                        onClose={ (): void => setShowDeleteConfirmationModal(false) }
+                        type="negative"
+                        open={ showRoleDeleteConfirmation }
+                        assertionHint={ t("console:manage.features.roles.edit.basics.confirmation.assertionHint") }
+                        assertionType="checkbox"
+                        primaryAction="Confirm"
```

Comment:
Can we add i18n here?

Reply:
Fixed with https://github.com/wso2/identity-apps/pull/4224/commits/7ef98d2bba165eaa30ca8d2d644b624a57acfbed